ldaptive/src/test/java/org/ccci/idm/user/ldaptive/dao/LdaptiveUserDaoTest.java

```diff
@@ -0,0 +1,10 @@
+package org.ccci.idm.user.ldaptive.dao;
+
+import org.ccci.idm.user.dao.AbstractUserDaoTest;
+
+public class LdaptiveUserDaoTest extends AbstractUserDaoTest {
```

Comment:
it's a test class that inherits the tests from the AbstractUserDaoTest class. That's where I have the readOnly test defined at